### DISCUSSION TITLE
看看能不能发

### DISCUSSION THREAD

**Discussion by FADE-4869** [share121/inter-knot#4741] [崩坏：星穹铁道]
#

[Uploading 封面00216000.bmp…]()

https://github.com/user-attachments/assets/d1ecd37a-3958-4eef-8867-2bca58ac9e02
[Uploading 封面00216000.bmp…]()

**Comment by fqing8485** [comment]
绳网可以发视频？

**Reply by 206326402** [reply]
可以啊
